web/src/components/Header.test.js

```diff
@@ -23,7 +22,7 @@ describe('Header component', () => {
     expect(component).toMatchSnapshot();
   });
 
-  it('opens and closes the dropdown when clicked', () => {
+  it('opens and closes the dropdown when clickedgit ', () => {
```

Comment:
clickedgit?

Reply:
I bet I was trying to do something with `git` but I had the wrong window focused... 🤦‍♂